test/xpu/run_test_with_skip.py

```diff
@@ -1693,6 +1692,13 @@ def launch_test(test_case, skip_list=None, exe_list=None):
     "test_reference_numerics_extremal__refs_tanh_xpu_complex64",
     "test_reference_numerics_extremal_tanh_xpu_complex128",
     "test_reference_numerics_extremal_tanh_xpu_complex64",
+    "test_reference_numerics_extremal__refs_acos_xpu_complex64",
```

Comment:
What's the detailed root cause of the failure?

Reply:
When the output is complex and contains inf, the output of xpu and expected(numpy) are not exactly same, such as xpu is 0.7853981256484985-infj, and expect is 0.7853981852531433-infj.